libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -435,8 +431,8 @@ int RawSiteToSiteClient::readRespond(const std::shared_ptr<Transaction> &transac
   return readResponse(transaction, code, message);
 }
 
-int RawSiteToSiteClient::writeRespond(const std::shared_ptr<Transaction> &transaction, RespondCode code, std::string message) {
-  return writeResponse(transaction, code, std::move(message));
+int RawSiteToSiteClient::writeRespond(const std::shared_ptr<Transaction> &transaction, RespondCode code, const std::string& message) {
+  return writeResponse(transaction, code, message);
```

Comment:
Changing that properly would lead further than I would like in this PR (this leads to a call to `size_t OutputStream::write(const std::string& str, bool widen = false);`)